drivers/frequency/adf5611/adf5611.c

```diff
@@ -0,0 +1,979 @@
+/**
+ *   @file   adf5611.c
+ *   @brief  Implementation of adf5611 Driver.
+ *   @author Jude Osemene (jude.osemene@analog.com)
+ *******************************************************************************
+* Copyright 2024(c) Analog Devices, Inc.
+*
+* All rights reserved.
+*
+* Redistribution and use in source and binary forms, with or without
```

Comment:
please update the license to our newest one: https://github.com/analogdevicesinc/no-OS/commit/a5b4c108fde614407df71ee972c6f5f1842134a8